wo8uy7-d1e549/cod-LyXLXy_wo8uy7-d1e549.xml

```diff
@@ -1,590 +1,775 @@
+          <name ref="#nivaca">Nicolas Vaughan</name>
+          <resp>Transcription Editor</resp>
+          <resp>TEI Encoder</resp>
+        </respStmt>
+      </titleStmt>
+      <editionStmt>
+        <edition n="0.0.0-dev">
+          <title>Quaestio 10</title>
+          <date when="2020-09-10">September 10, 2020</date>
+        </edition>
+      </editionStmt>
+      <publicationStmt>
+        <authority>SCTA</authority>
+        <availability status="free">
+          <p>Published under a <ref target="https://creativecommons.org/licenses/by-nc-sa/4.0/">Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)</ref>
+          </p>
+        </availability>
+      </publicationStmt>
+      <sourceDesc>
+        <listWit>
+          <witness xml:id="L" n="cod-LyXLXy">Lyon 1495</witness>
+        </listWit>
+      </sourceDesc>
+    </fileDesc>
+    <encodingDesc>
+      <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2020-09-15" status="draft" n="0.0.1">
+          <p>Revised text and added missing breaks, margin notes, etc.</p>
+        </change>
+        <change when="2020-09-10" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#L" n="136-v"/>
+        <cb ed="#L" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="wo8uy7-d1e549"><!-- l1d3q10 -->
+        <head xml:id="wo8uy7-d1e549-Hd1e108">Quaestio 10</head>
+        <p xml:id="wo8uy7-d1e549-d1e111">
+          <note place="margin-left">A</note>
+          <note place="margin-left">Questio X</note>
+          <lb ed="#L" n="91"/>Ultimo circam istam
+          <lb ed="#L" n="92"/>distinctionem quaero. Utrum creatura
+          <lb ed="#L" n="93"/>rationalis sit ymago trinitatis. Quod non
+          <lb ed="#L" n="94"/>quia tanta est distinctio in ymagine.
+          <lb ed="#L" n="95"/>quantam est in illo cuius est ymago.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e124">
+          ¶ Sed tres
+          perso<lb ed="#L" n="96" break="no"/>ne diuine realiter distinguntur. ergo in ymagine
+          <lb ed="#L" n="97"/>sunt partes realiter distincte. sed creatura rationalis
+          <lb ed="#L" n="98"/>est simplex. non habens partes realiter distinctas. ergo
+          <lb ed="#L" n="99"/>non est ymago.
+        </p>
+       
+       
+       
+        <p xml:id="wo8uy7-d1e549-d1e135">
+          ¶ Ad oppositum beatus Augustinus
```

Comment:
is the pilcrow (and underlying UTF code) getting changed between systems. The browsers shows different looking pilcrows in the previous file and the changed file. I'd like to make sure we use the same pilcrow encoding as this will make us more consistent throughout